release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -88,7 +88,7 @@ glusterFS.
 . The underlying PV is resized.
 
 [[ocp-310-ceph-provisioner]]
-==== Ceph Provisioner
+==== Ceph Provisioner (Technology Preview)
```

Comment:
Need more information on this.

Reply:
Need more information on Local Storage too (line 112)

Reply:
The local storage and ephemeral storage topics are the same.

Reply:
The Ceph Provisioner feature did not make the 3.10 release. There is more information in the comments: https://bugzilla.redhat.com/show_bug.cgi?id=1568345

Reply:
Thank you, @tmorriso-rh! You saved the day :cat: that helps a ton!